winui/Kanban/getting-started.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Getting Started with WinUI Kanban Control
+
+This section provides a quick overview of how to get started with the WinUI Kanban control (SfKanban) for WinUI and a walk-through to configure the WinUI Kanban control in a real-time scenario. Follow the steps below to add WinUI Kanban control to your project.
+
+## Creating an application with WinUI Kanban
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).
+
+2. Add reference to the `Syncfusion.Kanban.WinUI` NuGet.
+
+3. Import the control namespace `Syncfusion.UI.Xaml.Kanban` in XAML or C# code.
+
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 5" %}
+
+<Window 
+    ...
+    xmlns:kanban="using:Syncfusion.UI.Xaml.Kanban">
+
+    <kanban:SfKanban x:Name="kanban"/>
+</Window>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="1 9 10" %}
+
+using Syncfusion.UI.Xaml.Kanban;
+. . .
+
+public sealed partial class MainWindow : Window
+{
+    public MainWindow()
+    {
+        this.InitializeComponent();
+        SfKanban kanban = new SfKanban();
+        this.Content = kanban;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
+## Populate WinUI Kanban item source
+
+Here are the steps to render kanban card items using the WinUI Kanban control with the respective `KanbanModel` class:
+
+* Create the ViewModel.
+* Bind the item source for Kanban.
+
+### Define the view model
+
+Create a view model class to set values for the properties listed in the `KanbanModel` class as shown in the following example code. Each `KanbanModel` instance represents a card in the Kanban control.
+
+{% highlight c# %}
```

Comment:
Set tab title to ViewModel

![image](https://github.com/user-attachments/assets/6267aaac-37f1-4a3c-9962-7ee92c846f4a)